docs/QuickStart.rst

```diff
@@ -5,11 +5,12 @@ Quick Start
 At CSCS (Daint, Tsa, Balfrin)
 -----------------------------
 
-To set up a Spack instance, clone the repository
+To set up a Spack instance, clone the repository using a specific Spack tag (latest ``SPACK_TAG=v0.20.1.0``).
+For ICON, the SPACK_TAG given in `icon/config/cscs/SPACK_TAG <https://github.com/C2SM/icon/blob/main/config/cscs/SPACK_TAG>`_ should be used.
```

Comment:
This only holds for our distribution.
Users at MCH sticking to icon-nwp have a different file naming scheme.